source/_integrations/enphase_envoy.markdown

```diff
@@ -685,6 +685,95 @@ The Envoy Metered in multiphase setup, sums the voltages of the phases measured
 
 In multiphase installations with batteries, in countries with phase-balancing grid meters, the battery will export to the grid on one phase the amount it lacks on another phase. This other phase pulls the missing amount from the grid, as if it is using the grid as a 'transport' between phases. Since the grid meter will balance the amount imported and exported on the two phases, the net result is zero. The Envoy multiphase net-consumption CTs, however, will report the amounts on both phases, resulting in too high export on one and too high import on the other. One may consider using the `lifetime balanced net energy consumption` which is the sum of grid import and export to eliminate this effect. This would require some templating to split these values into import and export values. Alternatively, use the `current net power consumption` or `balanced net power consumption` with a Riemann integral sum helper.
 
+## Technical Information
+
+This integration uses the [pyenphase library](https://pypi.org/project/pyenphase/) to collect all Envoy data. Depending on the version of Home Assistant you are running, this integration may be using different versions of the library.
```

Comment:
This is not relevant for the end-user. We want to make things less technical, not more.